src/page-multi-facility/MultiFacilityImpactDashboard.tsx

```diff
@@ -110,10 +117,20 @@ const MultiFacilityImpactDashboard: React.FC = () => {
   };
 
   useEffect(() => {
-    if (referenceFacilitiesEligible && newlyAddedReferenceData) {
+    if (
+      referenceFacilitiesEligible &&
+      newlyAddedReferenceData &&
+      !showSyncNoFacilities
+    ) {
       setReferenceDataModalOpen(true);
+    } else {
+      setReferenceDataModalOpen(false);
     }
-  }, [referenceFacilitiesEligible, newlyAddedReferenceData]);
+  }, [
+    referenceFacilitiesEligible,
+    newlyAddedReferenceData,
+    showSyncNoFacilities,
+  ]);
```

Comment:
This update is used to ensure the `SyncNewReferenceData` and `SyncNoUserFacilities` do not conflict.  This works, but it's a little jerky so I'm going to see if I can improve it.

Reply:
Is it less jerky if the check for `!showSyncNoUserFacilities` is in the definition of `newlyAddedReferenceData`? If it is, maybe it's better to name this `showSyncNewReferenceData` and add the `referenceFacilitiesEligible` check in it too?